google-cloud-pubsub/samples/topics.rb

```diff
@@ -290,6 +290,37 @@ def publish_messages_async_with_concurrency_control topic_id:
   # [END pubsub_publisher_concurrency_control]
 end
 
+def publish_messages_async_with_flow_control topic_id:
+  # [START pubsub_publisher_flow_control]
+  # topic_id = "your-topic-id"
```

Comment:
I would put this line a bit further down, just above the `topic = ...` line where it is used, because that's where we'd expect to see it in actual code.

Reply:
This is the style throughout these samples, so it will be best not to change it in this PR.

Reply:
grrr... well, I guess I'll open an issue.